### ISSUE TITLE
Yggtorrent and Flaresolverr, cookie issue (seem so)

### ISSUE THREAD

**Issue by eephyne** [Prowlarr/Prowlarr#709]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I try to add indexer Yggtorrent, with flaresolverr (mandatory), I got this error:

https://pastebin.com/mvSC7SKz

and flaresolverr log seems fine:

`2021-12-13T09:04:18+00:00 INFO REQ-91 Incoming request => POST /v1 body: {"maxTimeout":60000,"cmd":"request.get","url":"https://www3.yggtorrent.re/engine/search?category=all&name=&description=&file=&uploader=&sub_category=&do=search&order=desc&sort=publish_date","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36"}
2021-12-13T09:04:18+00:00 WARN REQ-91 Request parameter 'userAgent' was removed in FlareSolverr v2.
2021-12-13T09:04:23+00:00 INFO REQ-91 Cloudflare detected
2021-12-13T09:04:30+00:00 INFO REQ-91 Challenge solved
2021-12-13T09:04:31+00:00 INFO REQ-91 Response in 12.903 s
`

I updated flaresolverr with the recent update to solve issue, before that, flaresolver wasn't working.

I don't know if this is still related to flaresolverr or prowlarr/jackett, the message in jackett talk about bad cookie sent by flaresolverr.

### Environment

```markdown
- OS: Arch
- Prowlarr: Nightly 0.1.8.1234
- Docker Install: no
- Using Reverse Proxy: yes but issue is the same directly
- Browser: chrome
```

### What branch are you running?

Nightly

### Trace Logs?

https://pastebin.com/mvSC7SKz

[AB#2137](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2137)

**Comment by bakerboy448** [comment]
- info logs are not valid for bug reports
- Pastebin should not be used to share logs as they have a tendency to block \*Arr logs like that have here
- YGG is confirmed to be working by others; FlareSolverr v2.1 is likely needed


Service Unavailable and being blocked by cloudflare is not a bug by itself

Please use one of the support channels: [Reddit](https://www.reddit.com/r/prowlarr/) or  [Discord](https://prowlarr.com/discord) for support/questions.

**Comment by bakerboy448** [comment]
Duplicate of https://github.com/Prowlarr/Prowlarr/issues/698
